grafana-dashboard.json

```
@@ -202,7 +936,10 @@
     "list": [
       {
         "allValue": null,
-        "current": null,
+        "current": {
+          "text": "3b19e6a3-dee3-404e-bebc-99cf7f99b574",
+          "value": "3b19e6a3-dee3-404e-bebc-99cf7f99b574"
+        },
```

Grafana likes to save a value for variables even when they don't make sense to store, so I just manually `null`-ed it out.

```suggestion
        "current": null,
```